### PR TITLE
Mocha chai

### PR DESCRIPTION
**Pull request by Fresheyeball**

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -56,6 +59,8 @@ options    = {
 port       = 3333,
 server     = express(),
 
+
+// FUCK YOU GULP! DID THIS EVER WORK? NO IDEA
```
Comment (vrecan) [inline gulpfile.js:63]: Gulp only works well if you speak nicely

Comment (Fresheyeball) [inline gulpfile.js:63]: I tried being nice to it, I tried yelling at it, I tried hitting it...

File: tests/Presentable/Router.spec.purs [hunk 2]
```diff
@@ -2,10 +2,18 @@ module Main where
...
 import Test.QuickCheck
 
 main = do
-  describe $ do
-    trace "what"
-    quickCheck $ \n -> foozle n == "In Port Foozle " ++ (show n) ++ " eats you!"
\ No newline at end of file
+  describe "its a thing!" $ do
+    it "seriously its a thing" $ do
+      quickCheck $ \n -> foozle n == "In Port Foozle " ++ (show n) ++ " eats you!"
```
Comment (vrecan) [inline tests/Presentable/Router.spec.purs:13]: foozled again

Comment (Fresheyeball) [inline tests/Presentable/Router.spec.purs:13]: [Foozle](http://lparchive.org/Zork-Grand-Inquisitor/Update%202/)

**Pull request merged**